plugins/node/opentelemetry-instrumentation-knex/README.md

```diff
@@ -17,7 +17,7 @@ npm install --save @opentelemetry/instrumentation-knex
 
 ### Supported Versions
 
-- `>=0.10.0`
+- [`knex`](https://www.npmjs.com/package/knex) versions `>=0.10.0 <4`
```

Comment:
Should we also remove the constants file for instrumentation-knex and move the supported versions into instrumentation, similar to what was done in instrumentation-router?

Reply:
I support a cleanup phase to the code to make all `supportedVersions` consistent and applied few fixes here and there in this PR. But I want to postpone a thorough cleanup to a later PR dedicated to this improvement and keep this one focused on the READMEs.